docs/examples/tutorial/cython_tutorial/primes.py

```diff
@@ -0,0 +1,27 @@
+def primes(nb_primes: cython.int):
+    i: cython.int
+    p: cython.int[1000]
+
+    if nb_primes > 1000:
+        nb_primes = 1000
+
+    if not cython.compiled:
+        p = [0] * 1000
```

Comment:
I haven't tried it but does `p: cython.int[1000] = [0] * 1000` work? I know if potentially initializes memory unnecessarily, but is saves the `if not cython.compiled`

Reply:
It works, but it generates additional code (for loop to fill array with 0-s), and therefore the python example will not match the cython example